utility/importer/brave_importer.cc

```diff
@@ -315,3 +327,52 @@ void BraveImporter::ImportStats() {
 
   bridge_->UpdateStats(stats);
 }
+
+std::vector<uint8_t> ParseWalletSeed(const base::Value& ledger_state_json) {
+  const base::Value* wallet_seed_json = ledger_state_json.FindPathOfType(
+      {"properties", "wallet", "keyinfo", "seed"},
+      base::Value::Type::DICTIONARY);
+  if (!wallet_seed_json) {
+    LOG(ERROR) << "Failed to parse wallet seed from ledger state";
+    return std::vector<uint8_t>();
+  }
+
+  // The wallet seed is a 32-byte Uint8Array, encoded in JSON as a
+  // dictionary of integers.
+  std::vector<uint8_t> wallet_seed;
+  for (int i = 0; i < 32; i++) {
+    const base::Value* value_json = wallet_seed_json->FindKey(std::to_string(i));
+    if (!value_json) {
```

Comment:
This is a great check- because there was a corruption issue before where the seed would only be 1 byte (instead of 32). It would show up as `A`

Reply:
Before we went to BIP32, we used Niceware and had 16 word restore key (representing the seed). @LaurenWags or @kjozwiak can share those files, in case they are not an array of 32 (IIRC, seed was 24 bytes?)

Reply:
@bsclifton Would it be easier/more historically reliable to use `ledger.info.passphrase` from `session-store-1` instead of `properties.wallet.keyinfo.seed` from `ledger-state.json`?

Reply:
nah- I think seed would be the right one to read. @LaurenWags can you share one of the older 16 word restore keys with @garrettr?

Reply:
Sent DM via slack - will get @garrettr what he needs 😄